apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html

```diff
@@ -85,8 +92,12 @@ <h5>
                             {{localeData?.total_receipts}}
                             <div>
                                 <span class="d-inline-flex number-total">
-                                    {{receiptsSummaryData?.total?.symbol}}{{receiptsSummaryData?.total?.amount |
-                                    giddhCurrency}}/-
+                                    <!-- {{receiptsSummaryData?.total?.symbol}}{{receiptsSummaryData?.total?.amount |
+                                    giddhCurrency}}/- -->
+
+                                    <amount-field [amount]="receiptsSummaryData?.total?.amount" [currencySymbol]="receiptsSummaryData?.total?.symbol" [currencyCode]="false">
```

Comment:
Do check currencyCode is not passed but currencySymbol is provided which will not make it RTL for Gulf countries.

Reply:
done